src/main/resources/db/migration/V1_151__add_appointment_session_feedback.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TYPE no_session_reason_type AS ENUM ('POP_ACCEPTABLE', 'POP_UNACCEPTABLE', 'LOGISTICS');
+
+alter table appointment
+    add column did_session_happen boolean,
+    add column no_attendance_information text,
```

Comment:
appointment table will have more column now, is it possible to take this to a new table and use `appointmentId` as a foreign key of that table.

Reply:
Yes, @rgforsyth and I discussed this and agreed we would do this as a separate technical debt/maintenance task.